036-BGP/Coach/README.md

```diff
@@ -0,0 +1,57 @@
+
+# Networking Hackathon - Info for coaches
+
+## Deploying the environment
+
+Since this Hack is not about deploying VNets or NVAs, you can use a script to deploy the infrastructure that you will be working on. You will find a script called `bgp.sh` in the file resources supplied for this Hack, which you can run to deploy the environment. The script has been tested to run in Azure Cloud Shell. Note that the script takes around 1 hour to complete, and it requires certain dependencies to exist. Therefore, it is recommended to deploy the environment the day before the Hack.
+
+Participants some times struggle with the required dependencies, deploying this in your own subscription in advance might help during the Hack, so you might want to deploy this in your subscription the day before.
+
+Note that there are no licensing costs associated to the Cisco NVAs.
+
+## Before you start

```

Comment:
Move this up, make it the first section.

Reply:
Done!